### DISCUSSION TITLE
AWS key-pair 관리

### DISCUSSION THREAD

**Discussion by myrhymetree** [sixes-sense/back-end#61] [General]
<img width="1097" alt="image" src="https://github.com/sixes-sense/back-end/assets/94158097/2a69123c-9c4c-46dd-81f0-78f09b8684a9">

<img width="723" alt="image" src="https://github.com/sixes-sense/back-end/assets/94158097/b7309b99-1248-4990-bc57-800a968b9819">

노션 팀프로젝트 / AWS / AWS KEY 경로에 저장

**Comment by myrhymetree** [comment]
@kny3037 @bbbbooo @ChoMyeongHwan @taegeun-park0525 aws key-pair 공유했습니다.

**Comment by kny3037** [comment]
확인했습니다!

**Comment by bbbbooo** [comment]
확인했습니다
